_test/test_mpi/test_MESS_NAMES_factory.m

```diff
@@ -0,0 +1,68 @@
+classdef test_MESS_NAMES_factory< TestCase
+    %
+    % $Revision:: 833 ($Date:: 2019-10-24 20:46:09 +0100 (Thu, 24 Oct 2019) $)
```

Comment:
what is this Revision string?

Reply:
that's the string which is currently updated during every static distribution creation. Its nice to see where the file came from when using continuoys integration

Reply:
But `833` doesn't match any Git SHA; Git gives you full line-by-line providence for any file.

Reply:
That's if you use git. As a first time look its better to have it obvious. and if it is a distributive, not related to git?

Reply:
The `master` version of Horace+Herbert are now on GitHub. Be definition we *are* using Git.

Reply:
Changing the version number management is already a separate issue ticket; so don't worry about this for now.